code/modules/reagents/reagents/reagents_medical.dm

```diff
@@ -1690,6 +1690,40 @@ var/global/list/charcoal_doesnt_remove=list(
 	if(prob(10))
 		M.drowsyness = max(M.drowsyness, 2)
 
+/datum/reagent/priaxate
+	name = "Priaxate"
+	id = PRIAXATE
+	description = "Priaxate is a broad spectrum medication formulated for the unique biology of vox. While still effective on other species, some of its effects are less potent."
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#CD8471" //rgb: 205, 132, 113
+	density = 1.29
+	specheatcap = 0.72
+	custom_metabolism = 0.4
+
+/datum/reagent/priaxate/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+
+	if(M.getOxyLoss())
+		M.adjustOxyLoss(-3 * REM)
+	if(M.getBruteLoss())
+		M.heal_organ_damage(3 * REM, 0)
+	if(M.getFireLoss())
+		M.heal_organ_damage(0, 3 * REM)
+	if(M.getToxLoss())
+		M.adjustToxLoss(-3 * REM)
+
+/datum/reagent/priaxate/on_mob_life(var/mob/living/M, var/alien)
```

Comment:
Duplicate proc. Merge this one with the one above it.

Reply:
Done